### PR TITLE
Adapt cnf-app-mac-operator to operator-sdk v1.33.0

### PR DESCRIPTION
**Pull request by ramperher**
- Created new cnf-app-mac-operator folder and creating operator-sdk v1.33.0 skeleton with the commands included in README
- Rest of the work was mainly to move missed files to the project, check all files created, adapt Makefile to the new structure, and adapt Golang code to new controller-runtime version used
- webserver is no longer needed for cnf-app-mac-operator controller-manager since it now includes healthz endpoint, so reusing it
- main.go is now under cmd/main.go
- controllers folder is now under internal/controller

### CODE REVIEW

**Comment by ramperher** [comment]
I've launched this job and it has passed the installation of example-cnf, now it's running preflight: https://www.distributed-ci.io/jobs/dbdfc0b4-00e4-4316-b511-c3963935ae6b/jobStates?sort=date. Change can be reviewed in the meantime.

**Comment by ramperher** [comment]
Change is working and example-cnf execution and certification looks good, this is ready for review.

Again, similarly to the testpmd-lb-operator, most changes are because of refactoring made by operator-sdk v1.33.0.

**Comment by ramperher** [comment]
I've just rebased after merging https://github.com/openshift-kni/example-cnf/pull/53, change is ready for review, no more tests are expected to be launched

**Comment by ramperher** [comment]
PR build check failed, probably because the new images for example-cnf latest versions are still being built in https://github.com/openshift-kni/example-cnf/pull/53, I'll relaunch after some minutes

**Comment by ramperher** [comment]
Just making a change in the README, no more debug tests are needed for the moment, waiting for reviews.

File: cnf-app-mac-operator/go.mod
Comment (capolrik) [inline cnf-app-mac-operator/go.mod:15]: Do you know why there is a new require section?

**Review by capolrik — APPROVED** [review]

File: cnf-app-mac-operator/go.mod
Comment (ramperher) [reply cnf-app-mac-operator/go.mod:15]: no idea :/ this file is automatically created in fact

**Comment by ramperher** [comment]
I'll wait till tomorrow EOB to receive any more feedback, else I will merge it

**Review by manurodriguez — APPROVED** [review]
LGTM

**Pull request merged**
